dbscSNV.pm

```diff
@@ -51,7 +51,7 @@ limitations under the License.
   > head -n1 dbscSNV1.1.chr1 > h
 
   # GRCh38
-  > cat dbscSNV1.1.chr* | grep -v ^chr | sort -k5,5 -k6,6n | cat h - | bgzip -c > dbscSNV1.1_GRCh38.txt.gz
+  > cat dbscSNV1.1.chr* | grep -v ^chr | sort -k5,5 -k6,6n | cat h - | awk '$5 != "."' | bgzip -c > dbscSNV1.1_GRCh38.txt.gz
```

Comment:
Does the order matter? I would have put awk before grep. But otherwise you are only getting rid of the entries without locations?

Reply:
This is regarding the recent comment on the issue https://github.com/Ensembl/VEP_plugins/issues/146.
I was wondering about the ordering, which is slightly different from the one on the dbNSFP's plugin:
https://github.com/Ensembl/VEP_plugins/blob/release/95/dbNSFP.pm#L53

Reply:
It is working for the user so the order doesn't matter.